ghcid-ng/src/ghci/stdout.rs

```diff
@@ -69,20 +68,20 @@ impl GhciStdout {
         prompt_patterns: Option<&AhoCorasick>,
     ) -> miette::Result<Option<CompilationResult>> {
         let prompt_patterns = prompt_patterns.unwrap_or(&self.prompt_patterns);
-        let lines = self
+        let data = self
             .reader
             .read_until(
                 prompt_patterns,
                 WriteBehavior::NoFinalLine,
                 &mut self.buffer,
             )
             .await?;
-        tracing::debug!(lines = lines.len(), "Got data from ghci");
+        tracing::debug!(bytes = data.len(), "Got data from ghci");
 
         let mut result = None;
         if self.mode == Mode::Compiling {
             result = self
-                .get_status_from_compile_output(lines)
+                .get_status_from_compile_output(data.lines().last())
```

Comment:
In the next PR, I'll replace this with the GHC output parser to avoid this inefficient traversal, but this makes tests pass.